bioinf_tools.py

```diff
@@ -0,0 +1,85 @@
+""" Bioinformatics tools 1
+
+This script contains various functions that could be utilised 
+as bioinformatics tools by the user.
+Imported modules (not Python build-in): 
+filter_fastq_module, dna_rna_tools_module
+"""
+
+from add_modules.filter_fastq_module import length_control
+from add_modules.filter_fastq_module import gc_control
+from add_modules import dna_rna_tools_module
+from statistics import mean
+
+
+def filter_fastq(
+    seqs: dict,
+    gc_bounds: tuple or "num" = (0, 100),
+    length_bounds: tuple or "num" = (0, 2**32),
+    quality_threshold: "num" = 0,
+) -> dict:
+    """The function filters sequences
+    based on the specified values ​​of the sequence length,
+    its GC composition, and the threshold value
+    of the average quality of the read.
+
+    Sequences that do not meet the specified parameters are discarded.
+    By default: the allowed values ​​for GC composition
+    are within (0,100) range, for sequence length - (0, 2**32)
+    and average read quality is 0.
+    If one number is passed to the gc_bounds argument,
+    then it is considered that this is the upper bound.
+    The function returns a dictionary with sequences
+    that meet the specified parameters.
+    """
+
+    seqs_checked = {}
```

Comment:
Я бы назвал:
```suggestion
    seqs_filtered = {}
```
ты же не просто проверяешь